core/src/themes/ionic.functions.sizes.scss

```diff
@@ -0,0 +1,13 @@
+@use "sass:math";
+
+$browser-context: 16;
+
+/**
+ * Convert a pixels given value into rem
+ * 
+ * @param pixels - Value in pixels to be converted (i.e. px)
+ * @param context (optional) - Baseline value
+ */
+@function pxToRem($pixels, $context: $browser-context) {
+  @return calc($pixels / $context) * 1rem;
```

Comment:
Is there any difference between the use of `calc` here vs. the `math.div` function from `sass:math`? 

We have a similar utility for dynamic font and I was curious if there was a reason to use one approach vs. the other. 

I do think this utility is helpful to have 👍

Reply:
There's not, it's the same since sass has the fallback to use calc at the same way browser do. didn't notice the usage of the `math.div`, in order to keep consistency I updated it to use `math.div` :)